databricks/koalas/frame.py

```diff
@@ -6797,14 +6818,29 @@ def melt(self, id_vars=None, value_vars=None, var_name=None,
 
         column_index = self._internal.column_index
 
+        missing = [True for id_var in id_vars if id_var not in column_index]
+        if True in missing:
+            raise KeyError("The following 'id_vars' are not present"
+                           " in the DataFrame: {}"
+                           .format(id_vars))
```

Comment:
Shall we use `name_like_string` for `id_vars`? E.g., `.format([nam_like_string(id) for id in id_vars])`

Reply:
I understand.
but I have a question, 
If id_vars is ['A', 'Z'], should we raise all of id_vars ('A', 'Z') or only non-existent column ('Z')?

Reply:
Could you check pandas' behavior? and shall we follow it?

Reply:
Ok.
I checked result of pandas' behavior.
```
>>> pdf.melt(value_vars=['A', 'Z'])
  ...
KeyError: "The following 'value_vars' are not present in the DataFrame: ['Z']"
>>> pdf.melt(value_vars=[('X', 'A'),('X', 'Z')])
  ...
KeyError: "The following 'value_vars' are not present in the DataFrame: ['Z']"
```
I changed the codes of the result of that exception.